### ISSUE TITLE
Missing `default_auto_field` in `wagtail.contrib.postgres_search`

### ISSUE THREAD

**Issue by michael-yin** [wagtail/wagtail#7369]
### Issue Summary

If user create new project with Django command, they will have 

`DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'`

In settings.py, and if they use `postgres_search`, they will see

```
web_1  |   Your models in app(s): 'postgres_search' have changes that are not yet reflected in a migration, and so won't be applied.
web_1  |   Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

### Technical details

In other Wagtail builtin app, the `app.py` have 

```
default_auto_field = 'django.db.models.AutoField'
```

I think we might need add it to the `wagtail/contrib/postgres_search/apps.py` to resolve this.

Thx.

**Comment by nickmoreton** [comment]
Is it worth me submitting a PR for this in light of it been added to 2.15 milestone? not sure how much time I have available but happy to dive in.

**Comment by gasman** [comment]
@nickmoreton Sure! We'll get it sorted some time in the run up to the 2.15rc1 release (some time in early October), but if you can get it crossed off the list sooner than that, it would be much appreciated :-)

**Comment by nickmoreton** [comment]
Thanks taking a look now, I found some time :)

**Comment by gasman** [comment]
Fixed in #7489
